CHANGELOG.rst

```diff
@@ -16,6 +16,11 @@ Change Log
 Unreleased
 -----------
 
+[3.16.9]
+--------
+
+* Upgrade celery to 5.0.4
+
```

Comment:
Does this change need to be deployed? Usually you wouldn't need a new version number (or a changelog entry) when updating a dependency.